File: python/fizzbuzz.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

def fizzbuzz():
    for num in range(1, 100):
        if num % 3 == 0 and num % 5 == 0:
            print("fizzbuzz")
        elif num % 3 ==0:
            print("fizz")
        elif num % 5 == 0:
            print("buzz")
        else:
            print(num)
    

if __name__ == '__main__':
    fizzbuzz()

File: ruby/fizzbuzz.rb
#!/usr/bin/env ruby
# -*- coding: utf-8 -*- 

def standard_fizzbuzz
  (1..100).each do |num|
    if num % 3 == 0 && num % 5 == 0
      puts "fizzbuzz"
    elsif num % 3 == 0
      puts "fizz"
    elsif num % 5 == 0
      puts "buzz"
    else
      puts num
    end
  end
end

def shorter_fizzbuzz
  (1..100).each{|num| out = '';out << 'fizz' if num % 3 == 0; out << 'buzz' if num % 5 == 0; out = num if out == ''; puts out}
end

standard_fizzbuzz
shorter_fizzbuzz
